src/main/java/uk/ac/ebi/ena/webin/cli/service/handler/NotFoundErrorHandler.java

```diff
@@ -1,52 +0,0 @@
-/*
- * Copyright 2018-2021 EMBL - European Bioinformatics Institute
```

Comment:
I think we should continue to use s NotFoundErrorHandler (and DefaultErrorHander if needed). See previous comment.